src/commands/check.js

```diff
@@ -218,6 +220,47 @@ export const handler = catchAsyncError(async opts => {
         isUpdateFinished = true;
         break;
 
+      case 'commit':
+        updatedModules.push(outdatedModule);
+        setModuleVersion(name, to, packageJson);
+        delete config.ignore[name];
+
+        // Showing the list of modules that are going to be updated
+        console.log(
+          `\n${strong('These packages will be updated:')}\n\n` +
+          createUpdatedModulesTable(updatedModules) +
+          '\n'
+        );
+
+        const {indent} = detectIndent(packageSource);
+        writeFileSync(
+          packageFile,
+          // Adding newline to the end of file
+          `${JSON.stringify(packageJson, null, indent)}\n`
+        );
+
+        try {
+          if (existsSync('yarn.lock')) {
+            // Assume the user wants to use yarn
+            execSync('yarn', {stdio: 'inherit'});
+            execSync('git add package.json yarn.lock', {stdio: 'inherit'});
+          } else if (existsSync('package-lock.json')) {
+            // Use npm and package-lock.json
+            execSync('npm install', {stdio: 'inherit'});
+            execSync('git add package.json package-lock.json', {stdio: 'inherit'});
+          } else {
+            // Default to only using npm install and not including a lock file
+            execSync('npm install', {stdio: 'inherit'});
+            execSync('git add package.json', {stdio: 'inherit'});
+          }
+          execSync(`git commit -m "Upgrade ${name} from ${from} to ${to}"`,{stdio: 'inherit'});
+          // Clean the list of packages to be updated after updating and commiting 
+          updatedModules.splice(0, updatedModules.length);
+        } catch(err) {
+          console.error(err)
```

Comment:
I think we need to show more user-friendly errors e.g. ``console.error(`Error upgrading ${name} module: ${err.message}`)``